### DISCUSSION TITLE
My ZX Sizif build log.

### DISCUSSION THREAD

**Discussion by terenceang** [UzixLS/zx-sizif-512#7] [Show and tell]
https://www.terenceang.com/2021/08/28/building-a-zx-spectrum-clone-in-2021-begining/

**Comment by UzixLS** [comment]
Thanks for sharing your experience. I added link into readme.

**Reply by terenceang** [reply]
Thank you. I will continue with my blog, 
Part 4 - casing mods and fittings, speakers and Bluetooth tape input. (next, once the parts arrive)
Part 5 - Using the zx sizif 512k.
Part 6 - 3D printed power supply.
Part 7 - 3D printed ZX Spectrum+ casing

Once again, thank you. The sizif-512k has been an excellent project for me to build. I hope to see more builds of it.
